spring-cloud-app-broker-integration-tests/build.gradle

```diff
@@ -29,7 +29,7 @@ dependencies {
 
 	testImplementation "org.springframework.boot:spring-boot-starter-test"
 	testImplementation "io.rest-assured:rest-assured"
-	testImplementation "com.github.tomakehurst:wiremock-jre8-standalone:${wiremockVersion}"
+	testImplementation "com.github.tomakehurst:wiremock-standalone:${wiremockVersion}"
```

Comment:
We should take the opportunity to migrate to the new group id too `org.wiremock`

Reply:
Oh, cool, didn't realize they changed that too, changing it too.